extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -199,14 +202,97 @@ export class AmpGeo extends AMP.BaseElement {
       this.mode_ = mode.GEO_HOT_PATCH;
       this.country_ = trimmedCountryMatch[1];
     } else if (trimmedCountryMatch[0] === '' && !getMode(this.win).localDev) {
```

Comment:
nit: I'd prefer to handle one mode in one if statement block. Can we please change the logic to
```
else if (...) {
  this.mode_ = mode.GEO_HOT_PATCH;
} else if (trimmedCountryMatch[0] === '' && urls.geoApi) {
  this.mode_ = mode.GEO_API;
} else if (!getMode(this.win).localDev) {
  this.error_ = true;
}
....
```

Reply:
Done.

Reply:
Thanks!